pkg/bundle/bundle_client.go

```diff
@@ -0,0 +1,63 @@
+package bundle
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+)
+
+type BundleClient interface {
```

Comment:
As above, I don't see why this couldn't just be `Client`, Mostly it would be referenced as `bundle.Client` anyway.